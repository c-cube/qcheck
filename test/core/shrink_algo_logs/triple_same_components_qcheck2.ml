open QCheck2

let triple_same =
  Test.make ~name:"triples have same components"
    Gen.(triple int int int)
    (Log.shrinks Print.(triple int int int) @@
       fun (i,j,k) -> i=j || j=k)

let rand = Random.State.make [|1234|];;
try Test.check_exn ~rand triple_same with Test.Test_fail _ -> ()
