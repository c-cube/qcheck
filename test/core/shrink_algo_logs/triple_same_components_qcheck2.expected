fails (3367339764960236310, -4202639126704614816, 2724675603984413065)
fails (0, -4202639126704614816, 2724675603984413065)
fails (0, 2587213195390812545, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1293606597695406272, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 646803298847703136, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 323401649423851568, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 161700824711925784, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 80850412355962892, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 40425206177981446, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 20212603088990723, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 10106301544495361, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 5053150772247680, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2526575386123840, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1263287693061920, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 631643846530960, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 315821923265480, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 157910961632740, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 78955480816370, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 39477740408185, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 19738870204092, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 9869435102046, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 4934717551023, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2467358775511, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1233679387755, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 616839693877, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 308419846938, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 154209923469, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 77104961734, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 38552480867, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 19276240433, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 9638120216, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 4819060108, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2409530054, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1204765027, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 602382513, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 301191256, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 150595628, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 75297814, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 37648907, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 18824453, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 9412226, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 4706113, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2353056, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1176528, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 588264, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 294132, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 147066, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 73533, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 36766, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 18383, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 9191, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 4595, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2297, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1148, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 574, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 287, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 143, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 71, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 35, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 17, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 8, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 4, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 2, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1, 2724675603984413065)
holds (0, 0, 2724675603984413065)
fails (0, 1, 0)
holds (0, 0, 0)
